exporter/sapmexporter/exporter.go

```diff
@@ -104,6 +105,7 @@ func newSAPMTracesExporter(cfg *Config, set component.ExporterCreateSettings) (c
 // and the last experienced error if any translation or export failed
 func (se *sapmExporter) pushTraceData(ctx context.Context, td ptrace.Traces) error {
 	rss := td.ResourceSpans()
+	td.ResourceSpans().At(0).ScopeSpans().At(0).Spans().At(0).SetSpanID(pcommon.InvalidSpanID())
```

Comment:
Is this a debug line?

Reply:
That was something I added while testing. Thanks for catching it. Removed. 

Also, updated `sapm-proto` to v0.11.0.